interface/wx/datetime.h

```diff
@@ -1575,8 +1575,8 @@ class wxDateTime
         Returns the object corresponding to the current UTC time including the
```

Comment:
I would remove the UTC reference from here as well.

Reply:
Oops, yes, thanks, don't know how I missed this.